ServerCore/Pages/Submissions/Index.cshtml

```diff
@@ -11,11 +11,14 @@
 
 @{
     var mailtoUrl = "mailto:";
-    
+
     // to: Add the proper support agent as recipient
     mailtoUrl += Model.PuzzleState?.Puzzle.SupportEmailAlias ?? (Model.Event?.ContactEmail ?? "puzzhunt@microsoft.com");
     // cc: Cc the team's email address because what if this email account is a random personal one?
-    mailtoUrl += "?cc=" + Uri.EscapeDataString(Model.Team.PrimaryContactEmail);
+    if (Model.Team.PrimaryContactEmail != null)
```

Comment:
pretty sure this mail is only null for demo teams but this is harmless so no big deal.

Reply:
Okay, I just added it since I was getting errors while testing this with the default sample event.